frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx

```diff
@@ -36,7 +36,7 @@ export const StorageClassFormProvisoners: ExtensionSCProvisionerProp = Object.fr
         imageFeatures: {
           name: 'Image Features',
           hintText: 'Ceph RBD image features',
-          value: 'layering',
+          value: 'layering,deep-flatten,exclusive-lock,object-map,fast-diff',
```

Comment:
is this required for Default SC?

Reply:
Yes please check https://github.com/red-hat-storage/ocs-operator/pull/1512

Reply:
then why this change? do we need to create all custom storage classes with these values?

Reply:
Yes, all new storageclasses should have these values.

Reply:
ack, change is LGTM